data/scripts/convert-config-to-json.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
Is there a need to have this script in the tree, once the configuration files are converted to JSON?